### PR TITLE
formatDate

### PR DESCRIPTION
**Pull request by DmitryVolchokFE**

### CODE REVIEW

**Comment by DmitryVolchokFE** [comment]
I wanted to get recommendations on the code, I'm sure it's possible better.

File: src/formatDate.js [hunk 1]
```diff
@@ -8,7 +8,31 @@
...
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arrDate = date.split(fromFormat[3]);
+  const fromDate = {};
+  const resultArr = [];
+
+  fromDate[fromFormat[0]] = arrDate[0];
+  fromDate[fromFormat[1]] = arrDate[1];
+  fromDate[fromFormat[2]] = arrDate[2];
```
Comment (etojeDenys) [inline src/formatDate.js:17]: you can use the loop here

File: src/formatDate.js [hunk 2]
```diff
@@ -8,7 +8,31 @@
...
+    fromDate['YYYY'] = `20${fromDate['YY']}`;
+  } else {
+    fromDate['YYYY'] = `19${fromDate['YY']}`;
+  }
+
+  resultArr.push(
+    fromDate[toFormat[0]],
+    fromDate[toFormat[1]],
+    fromDate[toFormat[2]],
+  );
```
Comment (etojeDenys) [inline src/formatDate.js:33]: and here

**Review by etojeDenys — APPROVED** [review]
but your code looks very well

**Pull request closed without merging**